lib/context.js

```diff
@@ -94,10 +94,10 @@ module.exports = function ctx(compiler, options) {
   }
 
   context.rebuild = rebuild;
-  context.compiler.plugin('invalid', invalid);
-  context.compiler.plugin('run', invalid);
+  context.compiler.hooks.invalid.tap({name: 'CopyPlugin'}, invalid)
```

Comment:
I believe all the names should be 'webpack-dev-middleware', not 'CopyPlugin'

Reply:
Indeed